app/assets/stylesheets/_vocab_sheet.scss

```diff
@@ -150,15 +150,24 @@
   background: white;
 }
 
-.vocab_sheet_page_item .item_name {
+.vocab_sheet_page_item .item_name,
+.vocab_sheet_page_item .item_maori_name {
   margin-bottom: 5px;
+  margin-right: 0px !important;
   position: relative;
   z-index: 11;
+  height: 5em;
+  // white-space: pre;
```

Comment:
can these commented bits be removed if they're not going to be re-enabled please